src/types.ts

```diff
@@ -66,6 +66,12 @@ export type TxFileData = {
 
 export type Address = string
 
+export enum DerivationType {
```

Comment:
Perhaps a comment that it only applies to Trezor, with a link to a description of the derivation types.

Reply:
Added